src/Neo.VM/JumpTable/JumpTable.Compound.cs

```diff
@@ -91,12 +118,24 @@ public virtual void Unpack(ExecutionEngine engine, Instruction instruction)
             engine.Push(compound.Count);
         }
 
+        /// <summary>
+        /// Creates a new empty array with zero elements on the evaluation stack.
+        /// <see cref="OpCode.NEWARRAY0"/>
+        /// </summary>
+        /// <param name="engine">The execution engine.</param>
+        /// <param name="instruction">The instruction being executed.</param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public virtual void NewArray0(ExecutionEngine engine, Instruction instruction)
```

Comment:
Change to `NewNullArray` method or add it?

Reply:
This is the name of the opcode.